lib/rialto/etl/transformers/people/names.rb

```diff
@@ -55,15 +55,15 @@ def fullname_from_names(given_name:, middle_name: nil, family_name:)
           # rubocop:disable Metrics/MethodLength
           def name_variations_from_names(given_name:, middle_name: nil, family_name:)
             name_variations = []
-            name_variations << "#{family_name}, #{given_name}"
-            name_variations << "#{given_name} #{family_name}"
-            if given_name
+            if given_name&.is_a?(String)
```

Comment:
Can you add a comment here that says that "WoS sometimes returns 'true' rather than a name, and we need to filter that out?

Can we do `return [] unless given_name&.is_a?(String)`?